runtime/doc/various.txt

```diff
@@ -35,8 +35,6 @@ CTRL-L			Clear and redraw the screen.  The redraw may happen
 			Note: if you like to use <BS> for this, add this
 			mapping to your .vimrc: >
 				:map CTRL-V <BS>   CTRL-V <Del>
-<			See |:fixdel| if your <Del> key does not do what you
```

Comment:
The `<` was removed, but I'm not sure it was accidental (isn't only a `>` needed for one line of code in help docs?).